robots/preservation/validate_bag.rb

```diff
@@ -0,0 +1,27 @@
+module Preservation
+  # Robot for validating bag in the Moab object deposit area
+  class ValidateBag < Base
+    ROBOT_NAME = 'validate-bag'.freeze
+
+    def initialize(opts = {})
+      super(REPOSITORY, WORKFLOW_NAME, 'validate-bag', opts)
```

Comment:
I _think_ the intent is to use the `ROBOT_NAME` constant as the 3rd argument rather than reproducing the string here.  Alternatively, just reproduce the string in the one place the constant is referenced (line 20) and remove the constant.  Either way is fine by me, but mix-and-match feels weird.

Reply:
good catch

Reply:
ja - missed that.  fixed.